src/styles/HomepageItem.css

```diff
@@ -0,0 +1,54 @@
+.item {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 160px;
+  background-color: rgba(54, 37, 114, 0.518);
+  border-radius: 10px;
+  text-align: center;
+  padding: 10px;
+  margin: 5px;
```

Comment:
- [ ] Please remove the spacings around your item and the white background color and use a darker or lighter tone of your main theme color to match the design layouts in Behance. Also, please use the white font color for the company names. It would be better to read. And you should generate their icon rather than just this fixed icon. That would make it more lively. Please try to filter the icons and use grayscale or sth else to make them shadowy as in the design below. They should look bigger. Take attention to each item's layout in the design below.
![image](https://github.com/waelelsafty07/Metrics-webapp/assets/91486791/c65d5ae7-a5e9-42a0-9451-f289779de613)

> your home page: 
![image](https://github.com/waelelsafty07/Metrics-webapp/assets/91486791/ad6d769a-d649-4af0-9306-32a6d47eeed9)
>behance:
![image](https://github.com/waelelsafty07/Metrics-webapp/assets/91486791/7de0cd23-14cc-40e6-aaf4-4b3a5797ee5d)

- [ ] There should be only 2 main colors to give shadow to each list element. But you have approximately 5 different colors including font colors. Please fix it.

- [ Optional ] I recommend you add some numeric values under your companies as in the Behance that enhance the look of your website.